src/renderers/dom/client/eventPlugins/ChangeEventPlugin.js

```diff
@@ -142,7 +152,7 @@ if (ExecutionEnvironment.canUseDOM) {
   // IE9 claims to support the input event but fails to trigger it when
   // deleting text, so we ignore its input events
   isInputEventSupported = isEventSupported('input') && (
-    !('documentMode' in document) || document.documentMode > 9
+    !('documentMode' in document) || document.documentMode > 11
```

Comment:
`documentMode` does not exist in MS Edge. Also, it seems weird to assume this would somehow be fixed in this theoretical version 12, it doesn't seem right to have a special-case for a browser we know nothing about.